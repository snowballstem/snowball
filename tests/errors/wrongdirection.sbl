externals ( stem )
routines ( remove_suffix )

backwardmode (
  define remove_suffix as (
    [substring] among (
      'al' 'ance' 'ence' 'er' 'ic' (delete)
    )
  )
)

define stem as (
  remove_suffix
)
