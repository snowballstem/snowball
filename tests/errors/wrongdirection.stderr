errors/wrongdirection.sbl:13: routine 'remove_suffix' mis-used in string forward mode
